.vitest/extend.ts

```diff
@@ -23,7 +23,7 @@ expect.extend({
 						),
 				};
 
-			if (result !== undefined) {
+			if (arguments.length === 3) {
```

Comment:
Just wanting to learn the motivation as I've not really use arguments length much over explicit checks. What exactly does this do differently? Is it safer?

Reply:
It allows us to expect an `undefined` result.
With the current implementation this check will be ignored (because we check on undefined), by using arguments the check will also verify if the result in undefined.